storage.bs

```diff
@@ -203,6 +203,16 @@ anticipated that some APIs will be applicable to both <a>storage types</a> going
 <p class=XXX>This is expected to change; see
 <a href="https://privacycg.github.io/storage-partitioning/">Client-Side Storage Partitioning</a>.
 
+<p>To <dfn export>obtain a storage key for non-storage purposes</dfn>, given an <a>environment
```

Comment:
Note: I used the "run these steps" format for this instead of condensing it down into one line because the "run these steps" format will be helpful once we incorporate top-level site into this. If using the more concise version is preferred until then I'm happy to make the change in this PR.